src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -422,15 +453,18 @@ Type[] IInvocationBinder.GetParameterTypes(string methodName)
         // REVIEW: We can decide to move this out of here if we want pluggable hub discovery
         private class HubMethodDescriptor
         {
-            public HubMethodDescriptor(ObjectMethodExecutor methodExecutor)
+            public HubMethodDescriptor(ObjectMethodExecutor methodExecutor, IEnumerable<IAuthorizeData> policies)
             {
                 MethodExecutor = methodExecutor;
                 ParameterTypes = methodExecutor.MethodParameters.Select(p => p.ParameterType).ToArray();
+                Policies = policies;
```

Comment:
Make this `IList<IAuthorizeData>` (to ToArray() like we do with parameter types)